default.nix

```diff
@@ -42,14 +42,15 @@ let iosSupport = system == "x86_64-darwin";
       };
     };
 
-    appleLibiconvHack = self: super: {
+    forceStaticLibs = self: super: {
       darwin = super.darwin // {
         libiconv = super.darwin.libiconv.overrideAttrs (_:
           lib.optionalAttrs (self.stdenv.hostPlatform != self.stdenv.buildPlatform) {
             postInstall = "rm $out/include/libcharset.h $out/include/localcharset.h";
             configureFlags = ["--disable-shared" "--enable-static"];
           });
       };
+      zlib = super.zlib.override { static = true; };
```

Comment:
Yeah this should work hopefully! Ideally we can move more of these for libiconv, libintl, here as well in overlay form. Right now we default to static on Android + iOS for those, but there is no necessity for it - it's just more convenient in reflex-platform.

Reply:
Yeah I'd love to collaborate with you, @nh2 and others on a fully static nixpkgs (stage, so native or cross) for this purpose.

Reply:
And my axe